includes/elements/column.php

```diff
@@ -936,29 +948,36 @@ public function before_render() {
 		$has_background_overlay = in_array( $settings['background_overlay_background'], [ 'classic', 'gradient' ], true ) ||
 								  in_array( $settings['background_overlay_hover_background'], [ 'classic', 'gradient' ], true );
 
-		$column_wrap_classes = [ 'elementor-widget-wrap' ];
+		$is_legacy_mode_active = Plugin::instance()->is_legacy_mode_active;
+		$legacy_space = $is_legacy_mode_active ? ' ' : '';
+		$wrapper_attribute_string = $is_legacy_mode_active ? '_inner_wrapper' : '_widget_wrapper';
+
+		$column_wrap_classes = $is_legacy_mode_active ? [ 'elementor-column-wrap' ] : [ 'elementor-widget-wrap' ];
 
 		if ( $this->get_children() ) {
-			$column_wrap_classes[] = 'elementor-element-populated';
+			$column_wrap_classes[] = $legacy_space . 'elementor-element-populated';
```

Comment:
Why do you need this `$legacy_space`?

Reply:
There was a space that was removed when removing the wrappers. Needed to add it back for compatibility

Reply:
Remove it